### ISSUE TITLE
Public Lists 

### ISSUE THREAD

**Issue by braian87b** [StevenBlack/hosts#162]
Here I found some others lists, hope it be useful. I didn't check all.

http://sysctl.org/cameleon/hosts at http://sysctl.org/cameleon

# Securemecca.com list - Also blocks "adult" sites (pornography/gambling etc)
http://www.hostsfile.org/Downloads/hosts.txt
mirror: http://securemecca.com/Downloads/hosts.txt
http://www.hostsfile.org/hosts.html is the same as http://www.securemecca.com/hosts.html

https://veryhost.googlecode.com/files/windwos.txt interesting, is small and divided as ads, Google, Facebook, Dropbox, Flickr, Wikipedia, could be used as white-listing

Inmortal domains: http://malwaredomains.lehigh.edu/files/immortal_domains.txt
http://malwaredomains.lehigh.edu/files/justdomains
just domains, but if you look at https://mirror.cedia.org.ec/malwaredomains/ you will see a lot of files...
here are information about source of data, and dates, etc... http://malwaredomains.lehigh.edu/files/domains.txt
mirror: https://mirror.cedia.org.ec/malwaredomains/
mirror: http://mirror1.malwaredomains.com/files/

https://zeustracker.abuse.ch/blocklist.php?download=domainblocklist
https://s3.amazonaws.com/lists.disconnect.me/simple_tracking.txt
https://s3.amazonaws.com/lists.disconnect.me/simple_ad.txt

# hpHosts’s Ad and tracking servers Updated frequently, but has been known to block legitimate sites.
http://hosts-file.net/ad_servers.txt

# Mahakala list. Has been known to block legitimage domains including the entire .com range.
#http://adblock.mahakala.is/

# ADZHOSTS list. Has been known to block legitimate domains
#http://optimate.dl.sourceforge.net/project/adzhosts/HOSTS.txt

# Windows 10 telemetry list - warning this one may block windows update
#https://raw.githubusercontent.com/crazy-max/HostsWindowsBlocker/master/hosts.txt

# Quidsup's tracker list
https://raw.githubusercontent.com/quidsup/notrack/master/trackers.txt

https://raw.githubusercontent.com/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt
http://spam404bl.com/spam404scamlist.txt
http://malwaredomains.lehigh.edu/files/domains.txt

**Comment by braian87b** [comment]
Ok, these seems to work well, others seems to had false positives or are too big...

```

mkdir data/HostsFile.org
echo "http://www.hostsfile.org/Downloads/hosts.txt" > data/HostsFile.org/update.info

mkdir data/SecureMecca.com
echo "http://www.hostsfile.org/Downloads/hosts.txt" > data/SecureMecca.com/update.info

mkdir data/hpHosts-ads
echo "http://hosts-file.net/ad_servers.txt" > data/hpHosts-ads/update.info

mkdir data/hpHosts-malware
echo "http://hosts-file.net/emd.txt" > data/hpHosts-malware/update.info # 3.8 Mb

mkdir data/hpHosts-fraud
echo "http://hosts-file.net/fsa.txt" > data/hpHosts-fraud/update.info # 2.9 Mb

mkdir data/hpHosts-spam
echo "http://hosts-file.net/grm.txt" > data/hpHosts-spam/update.info

mkdir data/hpHosts-hijack
echo "http://hosts-file.net/hjk.txt" > data/hpHosts-hijack/update.info

mkdir data/hpHosts-misleadingmarketing
echo "http://hosts-file.net/mmt.txt" > data/hpHosts-misleadingmarketing/update.info

mkdir data/hpHosts-pharmacy
echo "http://hosts-file.net/pha.txt" > data/hpHosts-pharmacy/update.info

mkdir data/hpHosts-phishing
echo "http://hosts-file.net/psh.txt" > data/hpHosts-phishing/update.info # 3.8 Mb

mkdir data/hpHosts-warez
echo "http://hosts-file.net/wrz.txt" > data/hpHosts-warez/update.info

rm -rf data/hpHosts-fraud    # 2.9 Mb
rm -rf data/hpHosts-phishing # 3.8 Mb
rm -rf data/hpHosts-malware  # 3.8 Mb
./makeHosts
find . -name hosts -exec ls -lh {} \;

-rw-r--r-- 1 root root 1.6K Aug 17 03:56 ./data/tyzbit/hosts
-rw-r--r-- 1 root root 165K Aug 17 03:56 ./data/hpHosts-misleadingmarketing/hosts
-rw-r--r-- 1 root root 755K Aug 17 03:56 ./data/hpHosts-pharmacy/hosts
-rw-r--r-- 1 root root 88K Aug 17 03:56 ./data/hpHosts-warez/hosts
-rw-r--r-- 1 root root 895 Aug 17 03:56 ./data/StevenBlack/hosts
-rw-r--r-- 1 root root 58K Aug 17 03:56 ./data/yoyo.org/hosts
-rw-r--r-- 1 root root 340K Aug 17 03:56 ./data/someonewhocares.org/hosts
-rw-r--r-- 1 root root 2.2K Aug 17 03:56 ./data/hpHosts-hijack/hosts
-rw-r--r-- 1 root root 233 Aug 17 03:56 ./data/SpotifyAds/hosts
-rw-r--r-- 1 root root 709K Aug 17 03:56 ./data/HostsFile.org/hosts
-rw-r--r-- 1 root root 19K Aug 17 03:56 ./data/hpHosts-spam/hosts
-rw-r--r-- 1 root root 14K Aug 17 03:56 ./data/adaway.org/hosts
-rw-r--r-- 1 root root 37K Aug 17 03:56 ./data/malwaredomainlist.com/hosts
-rw-r--r-- 1 root root 1.7M Aug 17 03:56 ./data/hpHosts-ads/hosts
-rw-r--r-- 1 root root 709K Aug 17 03:56 ./data/SecureMecca.com/hosts
-rw-r--r-- 1 root root 473K Aug 17 03:56 ./data/mvps.org/hosts
-rw-r--r-- 1 root root 3.5M Aug 17 03:56 ./hosts
-rw-r--r-- 1 root root 7.6K Aug 17 03:56 ./extensions/social/hosts
-rw-r--r-- 1 root root 19K Aug 17 03:56 ./extensions/gambling/hosts
-rw-r--r-- 1 root root 169K Aug 17 03:56 ./extensions/porn/hosts
-rw-r--r-- 1 root root 3.5M Aug 17 03:56 ./alternates/social/hosts
-rw-r--r-- 1 root root 3.6M Aug 17 03:56 ./alternates/gambling-porn/hosts
-rw-r--r-- 1 root root 3.5M Aug 17 03:56 ./alternates/gambling-social/hosts
-rw-r--r-- 1 root root 3.6M Aug 17 03:56 ./alternates/porn-social/hosts
-rw-r--r-- 1 root root 3.5M Aug 17 03:56 ./alternates/gambling/hosts
-rw-r--r-- 1 root root 3.6M Aug 17 03:56 ./alternates/gambling-porn-social/hosts
-rw-r--r-- 1 root root 3.6M Aug 17 03:56 ./alternates/porn/hosts

```

**Comment by 07416** [comment]
Some of these domains are already included. Windows 10 "privacy tools" have caused issues for me, the store app wasn't able to update for example.

**Comment by braian87b** [comment]
@07416 Yes, happened same to me, but this three seems to be secure to be appended, I had no problems so far... 

    HostsFile.org
    SecureMecca.com
    hpHosts-*

But as I mentioned here https://github.com/StevenBlack/hosts/issues/161 would be great to add white lists of secure domains to people that wants to allow microsoft domains (for updates, etc) or google domains (to allow google analytics or google adwords, etc) or others. In that manner if some source adds those domains would be protected...
I am currently out of time busy with others projects, but in a future will try to add this feature (last time I have done the same that does this script, this script didn't exist at that time, and I use a bash script as example and modified to allow multiple whitelists.

**Comment by Atavic** [comment]
Good idea about caring about OpenWRT. 

I can tell you that securemecca and immortaldomains are good lists, as I used both for years.

**Comment by braian87b** [comment]
@Atavic What I do is to run this scripts on a remote server and copy the hosts file to a /var/www/vhosts/domain/subdomain location, in that way I could download the hosts file easily from a OpenWrt device running a hotplug script (on wan-up) or cron schedule with a simple wget call to /tmp/block.hosts

Great Idea, I just write add a page on the Docs:

[AutoUpdate-on-OpenWRT-based-box-(also-works-on-LEDE)](https://github.com/StevenBlack/hosts/wiki/AutoUpdate-on-OpenWRT-based-box-(also-works-on-LEDE))

**Comment by Atavic** [comment]
My opinions, as someone who prefers to risk some false positive than leave some holes opened...

http://sysctl.org/cameleon has around 21K entries, mostly trackers and ads.
Mecuremecca is long-lived and well known.
Veryhost is gone 404.

For the rest, hpHosts’s Ad and tracking servers and Mahakala list are pretty hardcore and that's a plus to me.

https://zeustracker.abuse.ch/ is short but deals a sensitive risk.

http://spam404bl.com/ is very volatile.

**Comment by mmcnl** [comment]
Does anyone know a way to include automatic updates from a source that just lists domains, one-per-line (ie they are not prefixed by `0.0.0.0`)? For example, http://malwaredomains.lehigh.edu/files/justdomains

**Comment by StevenBlack** [comment]
@mmcnl I hope to have this feature implemented soon.

**Comment by Jezza-x3** [comment]
@mmcnl You should be able to just copy/paste it into notepad or the likes and replace "0.0.0.0" with "" (nothing).
